app/src/main/res/values-fr/help.xml

```diff
@@ -0,0 +1,247 @@
+        C’est la seule donnée necessaire pour récupérer votre portefeuille ultérieurement
+        et obtenir l’accès à vos fonds.
+        Il est très imprtant de la garder sécurisée et confidentielle, car elle donne
+        <em>à quiconque</em> le contrôle total de vos fonds !</p>
+        <p>Si vous perdez le mot de passe de votre portefeuille, vous pouvez toujours récupérer votre
+        portefeuille avec la phrase Mnémonique.</p>
+        <p>Il n’y a aucun moyen de récupérer votre phrase Mnémonique, si vous la perdez tous vos fonds
+        seront perdus ! La phrase Mnémonique ne peut pas non plus être changée, et si elle est dérobée
+        ou compromise, vous devrez transférer vos fonds vers un nouveau portefeuille (avec une nouvelle
+        phrase Mnémonique). Par conséquent, il vaut mieux que vous sauvegardiez votre phrase Mnémonique
+        en l’écrivant et en la conservant à de <em>multiples</em> endroits sécurisés.</p>
+    ]]></string>
+
+    <string name="help_create_seed"><![CDATA[
+        <h1>Créer un Portefeuille - Phrase Mnémonique</h1>
+        <p>Si vous avez déjà une adresse Monero et que vus voulez reécupérer vos transactions depuis la chaïne de blocs !</p>
+        <p>Entrez un nom et un mot de passe unique pour le portefeuille.
+        Le mot de passe permet de sécuriser les données de votre portefeuille sur votre appareil.
+        Utilisez un mot de passe fort - mieux, utilisez une phrase secrète.</p>
+        <p>Entrez votre phrase Mnémonique dans le champ \"Phrase Mnémonique\".<p>
+        <p>Si vous connaissez le numéro du bloc de votre première transaction à cette adresse, entrez-le dans le
+        champ \"Hauteur de restauration\" - en le laissant vide la <em>totalité</em> de la chaîne de blocs sera scannée
+        à la recherche des transactions appartenant à votre adresse. Cela prend <em>beaucoup</em> de temps.</p>
+    ]]></string>
+
+    <string name="help_create_keys"><![CDATA[
+        <h1>Créer un Portefeuille - Clefs</h1>
+        <p>Si vous récupérez votr portefeuille en utilisant vos clefs !</p>
+        <p>Entrez un nom et un mot de passe unique pour le portefeuille.
+        Le mot de passe permet de sécuriser les données de votre portefeuille sur votre appareil.
+        Utilisez un mot de passe fort - mieux, utilisez une phrase secrète.<p>
+        <p>Entrez votre adresse Monero dans le champ \"Adresse Publique\" et remplissez \"Clef d’Audit\" et \"Clef de dépense\".</p>
+        <p>Si vous connaissez le numéro du bloc de votre première transaction à cette adresse, entrez-le dans le
+        champ \"Hauteur de restauration\" - en le laissant vide la <em>totalité</em> de la chaîne de blocs sera scannée
+        à la recherche des transactions appartenant à votre adresse. Cela prend <em>beaucoup</em> de temps.</p>
+    ]]></string>
+
+    <string name="help_create_view"><![CDATA[
+        <h1>Créer un Portefeuille - Audit</h1>
```

Comment:
maybe make this "Portefeuille d'Audit" to match the GUI ?

Reply:
sure.